packages/react-composites/src/composites/CallComposite/components/SidePane/useVideoEffectsPane.tsx

```diff
@@ -52,32 +51,27 @@ export const useVideoEffectsPane = (
     );
   }, [closePane, /* @conditional-compile-remove(video-background-effects) */ locale.strings, mobileView]);
 
-  const [dismissedVideoEffectsError, setDismissedVideoEffectsError] = useState<DismissedError>();
-  const [activeVideoEffect, setActiveVideoEffect] = useState<ActiveVideoEffect>();
-  const onDismissVideoEffectError = useCallback((error: AdapterError) => {
-    setDismissedVideoEffectsError(dismissVideoEffectsError(error));
-  }, []);
-  const latestVideoEffectError = useSelector(videoBackgroundErrorsSelector);
-  const activeVideoEffectError = useCallback(() => {
-    if (
-      latestVideoEffectError &&
-      (!dismissedVideoEffectsError || latestVideoEffectError.timestamp > dismissedVideoEffectsError.dismissedAt) &&
-      (!activeVideoEffect || latestVideoEffectError.timestamp > activeVideoEffect.timestamp)
-    ) {
-      return latestVideoEffectError;
-    }
-    return undefined;
-  }, [dismissedVideoEffectsError, latestVideoEffectError, activeVideoEffect]);
+  /* @conditional-compile-remove(video-background-effects) */
+  const latestVideoEffectError = latestErrors.find((error) => error.type === 'unableToStartVideoEffect');
 
   const onRenderContent = useCallback((): JSX.Element => {
     return (
       <VideoEffectsPaneContent
-        onDismissError={onDismissVideoEffectError}
-        activeVideoEffectError={activeVideoEffectError}
-        setActiveVideoEffect={setActiveVideoEffect}
```

Comment:
We have removed set active video effects, don't we need that information for the pane anymore?